apps/user-profile/src/app/pages/settings-page/settings-alternative-passwords/settings-alternative-passwords.component.ts

```diff
@@ -123,9 +119,11 @@ export class SettingsAlternativePasswordsComponent implements OnInit {
     this.attributesManagerService
       .getUserAttributeByName(this.userId, `urn:perun:user:attribute-def:def:altPasswords:einfra`)
       .subscribe((att) => {
-        this.altPasswordsAttribute = att;
         if (att.value) {
```

Comment:
Maybe we should also check here that the attribute value is of correct type. Also please use some meaningful varibale name instead of `foo`.